.devops/pagopa-code-review-pipelines.yml

```diff
@@ -34,7 +34,7 @@ resources:
     - repository: checkout-be-mock
       type: github
       name: pagopa/pagopa-checkout-be-mock
-      ref: master
+      ref: CHK-2511-added-NPG-unauthorized-status-case
```

Comment:
Please, restore the master branch

Reply:
Fixed with 5e7731efe146916dff616bd124ee4361ee059472